CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Fixed
+- We upgraded minimum sdk version to 33
+
+## [7.0.3] - 2023-08-08
+### Fixes
+- We fixed distorted splash screen issue for android
```

Comment:
is this fix part of the changes?

Reply:
This one is not part of the changes, this is for 7.0.3, I realized it wasn't there

Reply:
I noticed that the CHANGELOG is not updated, this is the previous version note